vagrant/Vagrantfile

```diff
@@ -2,53 +2,73 @@
-    config.vm.hostname = 'vagrant-baragon-service'
-    config.vm.network :private_network, ip: '192.168.33.20'
-
-    # This will enable the vagrants to connect to the zookeeper server on you local machine, comment out to only use the zookeeper server on the baragon-service machine
-    config.vm.network "forwarded_port", guest: 2181, host: 2181
+  config.cache.auto_detect = true if Vagrant.has_plugin?("vagrant-cachier")
 
-    config.vm.provision "shell", path: "provision-service.sh"
+  config.vm.define "service", autostart: true do |build|
+    build.vm.hostname = 'vagrant-baragon-service'
+    build.vm.network :private_network, ip: '192.168.33.20'
+    build.vm.network "forwarded_port", guest: 2181, host: 2181
+    build.vm.provision "shell", path: "provision-service.sh"
   end
 
-  config.vm.define "agent", autostart: true do |config|
-    config.vm.hostname = 'vagrant-baragon-agent'
-    config.vm.network :private_network, ip: '192.168.33.21'
-    config.vm.provision "shell" do |s|
+  config.vm.define "agent", autostart: true do |build|
+    build.vm.hostname = 'vagrant-baragon-agent'
+    build.vm.network :private_network, ip: '192.168.33.21'
+    build.vm.provision "shell" do |s|
       s.path =  "provision-agent.sh"
       s.args =  "vagrant"
-     end
+    end
   end
 
   # Used to spin up a additional agents for multi-agent testing
   %w(2 3 4 5).each do |x|
-      config.vm.define "agent#{x}", autostart: false do |config|
-        config.vm.hostname = "vagrant-baragon-agent#{x}"
-        config.vm.network :private_network, ip: "192.168.33.2#{x}"
-        config.vm.provision "shell" do |s|
-          s.path =  "provision-agent.sh"
-          s.args =  "vagrant#{x}"
-         end
-      end
+    config.vm.define "agent#{x}", autostart: false do |build|
+      build.vm.hostname = "vagrant-baragon-agent#{x}"
+      build.vm.network :private_network, ip: "192.168.33.2#{x}"
+      build.vm.provision "shell" do |s|
+        s.path =  "provision-agent.sh"
+        s.args =  "vagrant#{x}"
+       end
+    end
+  end
+
+  # baragon-develop base image
+  config.vm.define "base_image", autostart: false do |build|
+    build.vm.box = "opscode_ubuntu-14.04_provisionerless"
+    build.vm.box_url = "http://opscode-vm-bento.s3.amazonaws.com/vagrant/virtualbox/opscode_ubuntu-14.04_chef-provisionerless.box"
+    build.vm.network :private_network, ip: "192.168.33.30"
+
+    build.berkshelf.berksfile_path = 'Berksfile'
+    build.berkshelf.enabled = true
+    build.omnibus.chef_version = :latest
+
+    build.vm.provision :shell do |shell|
+      shell.inline = 'test -f $1 || (sudo apt-get update -y && touch $1)'
+      shell.args = '/var/run/apt-get-update'
+    end
+    build.vm.provision :chef_solo do |chef|
+      chef.log_level = :info
+      chef.run_list = ["java"]
+      chef.json = {:java => {:install_flavor => "oracle", :jdk_version => 7, :oracle => {:accept_oracle_download_terms => true}, :set_default => true}}
```

Comment:
the `build` recipe in the cookbook doesnt provide this? might be worth adding to the cookbook if not